js/npm/etherscan/test/smoke.spec.js

```diff
@@ -14,21 +14,8 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
-import 'babel-polyfill/dist/polyfill.js';
-import es6Promise from 'es6-promise';
-es6Promise.polyfill();
+const assert = require('assert');
 
-const isNode = typeof global !== 'undefined' && typeof global !== 'undefined';
-const isBrowser = typeof self !== 'undefined' && typeof self.window !== 'undefined';
-
-if (isBrowser) {
-  require('whatwg-fetch');
-}
-
-if (isNode) {
-  global.fetch = require('node-fetch');
-}
-
-import ShapeShift from './3rdparty/shapeshift';
-
-export default ShapeShift;
+assert.doesNotThrow(() => {
```

Comment:
I think it's just not to introduce dependencies as this a the built-in `assert` package